setup.cfg

```diff
@@ -1,10 +1,66 @@
 [metadata]
-package_name = jwst
-description = JWST Pipeline
-author = STScI
-license = BSD
-edit_on_github = False
-github_project = spacetelescope/jwst
+name = jwst
+description = Python library for science observations from the James Webb Space Telescope
+long_description = file: README.md
+long_description_content_type = text/markdown
+author = JWST Pipeline developers
```

Comment:
This is not the whole JWST pipeline (that includes all of DMS). We need to be careful to qualify every instance of "pipeline" with "calibration".